Bottom: e2890710c970dd207d6637b2e82f3ca0c5c38f25
Top:    b3eb5fda07302d707d4cd9d8de8c0c0c6d1e785a
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-05-11 15:38:59 +0800

Refresh of vhost_net-add-interrupt

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index ac24979..9740534 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -415,7 +415,7 @@ static void handle_tx(struct vhost_net *net)
 		}
 	}
 out:
-	vhost_check_coalesce_and_signal(dev, vq);
+	vhost_check_coalesce_and_signal(vq->dev, vq);
 	mutex_unlock(&vq->mutex);
 }
 
@@ -639,7 +639,7 @@ static void handle_rx(struct vhost_net *net)
 		}
 	}
 out:
-	vhost_check_coalesce_and_signal(dev, vq);
+	vhost_check_coalesce_and_signal(vq->dev, vq);
 	mutex_unlock(&vq->mutex);
 }
